src/discordtipbot/discordbot.cr

```diff
@@ -374,7 +391,7 @@ class DiscordBot
     when false
       reply(msg, "**ERROR**: There was a problem trying to withdraw. Please try again later. If the problem persists, please contact the dev for help in #{@config.prefix}support")
     when true
-      reply(msg, "Successfully withdrew **#{amount} #{@config.coinname_short}** to **#{address}**")
+      reply(msg, "Pending withdrawal of **#{amount} #{@config.coinname_short}** to **#{address}**. *Processing shortly*" + CURSOR)
```

Comment:
I kind of think "processing shortly" should be removed.

Reply:
Like completely get rid of it? Not sure if people will really understand what's happening